ffmpeg/transcoder.c

```diff
@@ -439,6 +439,12 @@ int lpms_transcode(input_params *inp, output_params *params,
     if (ret < 0) {
       return ret;
     }
+  } else if (inp->reopendemux) {
+      free_input(&h->ictx);
+      ret = open_input(inp, &h->ictx);
```

Comment:
This will re-initialize NVDEC and add delay - could we just free audio context, and call `open_audio_decoder()`? I imagine that streams with some video-only segments are pretty common.

Reply:
* So it's unlikely to be called mid-stream, but it still will be called if first segment doesn't have audio, and next segment has audio, is that right?
* Right, it's probably not so easy to inject audio demuxer dynamically, if possible at all. I wonder how would we address such issue withing low latency streaming workflow?

Reply:
> So it's unlikely to be called mid-stream, but it still will be called if first segment doesn't have audio, and next segment has audio, is that right?

Right.

> Right, it's probably not so easy to inject audio demuxer dynamically, if possible at all. I wonder how would we address such issue withing low latency streaming workflow?

Yeah, I tried to inject audio demuxer dynamically, but couldn't find a reasonable path without reopening the segment.
open_audio_decoder() returns "No audio stream found in input" message.